charts/aws-ebs-csi-driver/templates/metrics.yaml

```diff
@@ -16,22 +16,24 @@ spec:
       targetPort: 3301
   type: ClusterIP
 ---
-{{- if (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") -}}
+{{- if or (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") (.Values.controller.serviceMonitor.forceEnable) }}
```

Comment:
Nit: parenthesis not needed around `forceEnable` value

Reply:
Done.